3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/enrichments.scala

```diff
@@ -58,11 +61,11 @@ trait ParseableEnrichment {
    * @return The JSON or an error message, boxed
    */
   def isParseable(config: JValue, schemaKey: SchemaKey): ValidatedNelMessage[JValue] = {
-    if (schemaKey == supportedSchemaKey) {
+    if (supportedSchema matches schemaKey) {
       config.success
     } else {
       ("Schema key %s is not supported. '%s' enrichments must have schema %s.")
```

Comment:
I think "enrichment" not "enrichments", as would never be plural for a single user.